prow/config/prow-config-documented.yaml

```diff
@@ -540,10 +540,6 @@ plank:
     build_cluster_status_file: ' '
 
     # DefaultDecorationConfigEntries is used to populate DefaultDecorationConfigs.
-    # Each entry in the slice specifies Repo and Cluster regexp filter fields to
-    # match against jobs and a corresponding DecorationConfig. All entries that
-    # match a job are used. Later matching entries override the fields of earlier
-    # matching entries.
```

Comment:
I don't think we should remove this. If we do, this information is not included in the documented config file at all.
Since this is important info I don't think redundancy in the godocs hurts.

Reply:
done